src/sdk/analytics/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react'
+import { useAnalyticsEvent } from '@vtex/store-sdk'
+import type { FC } from 'react'
+
+export const AnalyticsProvider: FC = ({ children }) => {
```

Comment:
I think it'd be better if we renamed this component to something like `AnalyticsHandler`. It doesn't provide a context, which is when we usually name things as providers in React. I think leaving it this way may confuse people :)

Reply:
That makes a lot of sense. Done at https://github.com/vtex-sites/base.store/pull/53/commits/d1f33648e138fbb01faeed277229284a22e3405a

Reply:
@tlgimenes here is where the `AnalyticsProvider` was renamed to `AnalyticsHandler`